src/test/kotlin/com/forgerock/uk/openbanking/framework/constants/Constants.kt

```diff
@@ -1,7 +1,3 @@
 package com.forgerock.uk.openbanking.framework.constants
 
-const val REDIRECT_URI = "https://google.com"
```

Comment:
Adding a comment to hopefully make the intention of the PR clearer:

This is the value that was being used previously, it could simply have been updated to a value in our "Automating Testing" SSA i.e. https://www.google.co.uk

However, this is configuration mixed in with the code. So this PR also moves REDIRECT_URI into Config.